### PR TITLE
add grin deposit history

### PR DESCRIPTION
**Pull request by shiqicao**
Enable mw deposit history(un-styled), and add "Download" button.

### CODE REVIEW

File: public/templates/funds/_deposit_coin_history.html [hunk 1]
```diff
@@ -19,6 +20,7 @@ <h2 class="panel-title">{{ "funds.deposit_history.title" | t }}</h2>
     <td>{{deposit.fee}}</td>
     <td>{{deposit.confirmations}}</td>
     <td>{{"funds.deposit_history." + deposit.aasm_state | t}}</td>
+    <td><a href="{{deposit.response_file_url}}">Donwlaod</a></td>
```
Comment (jordansaints) [inline public/templates/funds/_deposit_coin_history.html]: `Download`

File: public/templates/funds/_deposit_mw_coin_history.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<br />
```
Comment (jordansaints) [inline public/templates/funds/_deposit_mw_coin_history.html]: ?? lol trying to ignore the markup 😝

File: public/templates/funds/_deposit_coin_history.html [hunk 3]
```diff
@@ -19,6 +20,7 @@ <h2 class="panel-title">{{ "funds.deposit_history.title" | t }}</h2>
...
   </tr>
   <tr ng-show="dhCtrl.noDeposit()"><td colspan="6"><span class="help-block text-center">{{ "funds.deposit_history.no_data" | t }}</span></td></tr>
```
Comment (jordansaints) [inline public/templates/funds/_deposit_coin_history.html]: should probably update `colspan="6"` to `colspan="7"` to match the new column count

File: public/templates/funds/deposit_mw_coin.html [hunk 4]
```diff
@@ -3,3 +3,5 @@ <h2 class="panel-title">{{"funds.deposit_coin.title" | t:currencyTranslationLoca
     <input type = "file" onchange="angular.element(this).scope().fileChange(this)"/>
     <button ng-click="uploadFile()" >upload me</button>
 </div>
+
+<ng-include src="'/templates/funds/_deposit_mw_coin_history.html'"></ng-include>
```
Comment (jordansaints) [inline public/templates/funds/deposit_mw_coin.html]: does the string need to be double quoted? (double quotes + single quotes?)

File: public/templates/funds/_deposit_coin_history.html (hunk 3 above)
Comment (shiqicao) [reply public/templates/funds/_deposit_coin_history.html]: nice catch!

File: public/templates/funds/_deposit_mw_coin_history.html (hunk 2 above)
Comment (shiqicao) [reply public/templates/funds/_deposit_mw_coin_history.html]: we don't need it, removed

**Pull request merged**